portworx-manual/temp-px-perf/px-snapshottest.sh

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env bash
+

```

Comment:
This empty lines seem to be causing errors when running this script. What I get:

```
$ sh px-snapshottest.sh 
: command not found line 2: 
: command not found line 4: 
: command not found line 6: 
volumesnapshot.volumesnapshot.external-storage.k8s.io "px-kafka-snapshot" created
: command not found line 8: 
Verifying volume snapshot creation
: command not found line 10: 
error: the server doesn't have a resource type "volumesnapshot\r"
: command not found line 12: 
NAME                                                       AGE
k8s-volume-snapshot-38515a1a-b993-11e9-86cf-8ae310e7e8ae   2s
: command not found line 14: 
Deleting kafka broker stateful set and PVC
: command not found line 16: 
Error from server (NotFound): statefulsets.apps "px-cluster-kafka\r" not found
Error from server (NotFound): persistentvolumeclaims "data-px-cluster-kafka-0\r" not found
Error from server (NotFound): persistentvolumeclaims "data-px-cluster-kafka-1\r" not found
Error from server (NotFound): persistentvolumeclaims "data-px-cluster-kafka-2\r" not found
No resources found.
No resources found.
No resources found.
: command not found line 24: 
 Creating new PVC definition from the snapshot.
: command not found line 26: 
persistentvolumeclaim "data-px-cluster-kafka-0" created
No resources found.
: command not found line 29: 
Deploying new stateful set backend by the restored snapshot PVC will be automatically handled by the Strimzi Operator
: command not found line 31: 
: command not found line 33: 
....
```

Reply:
Ran a `dos2unix` on a file. It had window `/r` chars